.gitignore

```diff
@@ -66,7 +66,6 @@ m4/
 make.sh
 missing
 mkinstalldirs
-po/
```

Comment:
Are we okay with re-introducing the `po/` directory again? If not what would be a preferred location in the repository?